sdks/go/container/license_script.sh

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+output_dir=third_party_licenses
+# remove output_dir if existing
+if [ -d "$output_dir" ]; then rm -rf $output_dir; fi
+
+# get go-licenses and run
+go get github.com/google/go-licenses
+$GOPATH/bin/go-licenses save "github.com/apache/beam/sdks/go/pkg/beam/" --save_path="$output_dir"
```

Comment:
This line returns `not found` error when run with Jenkins. When I test with my machine, `$GOPATH/bin/go-license` worked.
I tried with `go-licenses`, `$GOPATH/bin/go-license`, `/usr/bin/go/bin/go-licenses` but no one worked. All returned `not found` error. [log](https://builds.apache.org/job/beam_PreCommit_Go_Commit/6020/console)
How do we run a Go package within Jenkins? Can we use the same script to run both locally and Jenkins? And in case users want to customize it, it should be able to run at users' machine as well.

Reply:
cc: @lostluck

Reply:
The Go SDK isn't currently up on all the best practices for Go. In particular, what you're running on your machine is probably not what Jenkin's is pulling down due to how versioning isn't being handled at present. It's not currently a priority to spend effort to resolve this state of affairs. It will not happen until after SDF and Beam Schema's have been added to the SDK later this year.

I do not recommend spending time *now* to do this, but please keep sufficient notes so that we can resolve the Go situation later on.

Reply:
As long as we release Go sdk docker images, we should include licenses to avoid legal issues. If we don't want to work on this now, we should not release Go images. 

In addition, effort is already taken for this work. We only need to find a way to run the go package within Jenkins. Working on it later requires more effort given current status. (revisiting it to refresh from memory, updating documents, reviewing docs and code again later etc.)

cc: @aaltay

Reply:
I agree with @Hannah-Jiang. We can either pause releasing go container images or make the changes now to include these files in the released containers.

I also agree with @lostluck. If versioning will make this much easier later we can defer working on this to save effort.

@lostluck - would it be fine to pause releasing go containers until versioning story is in? Would this impact beam users? (Note, this would not prevent folks from build containers of their own.)

Reply:
The instructions currently indicate users should build their own. The Go SDK Jenkins runs only use their own built images so testing will continue to work. The code doesn't currently make reasonable default choices for built images.

Reply:
Sounds good. 

@Hannah-Jiang - could we remove Go containers from Beam release instructions and add a link to JIRA that needs to be resolved before starting that process again?